### ISSUE TITLE
Unit test failure

### ISSUE THREAD

**Issue by Myoldmopar** [NREL/EnergyPlus#7058]
Issue overview
--------------
A unit test (`EnergyPlusFixture.PackagedTerminalHP_VSCoils_Sizing`) intermittently fails.  The failure rate is probably like 20%.  The platform is **always** Mac.  Anyway, something isn't being cleared in a `clear_state`.  
 
Example failure: http://nrel.github.io/EnergyPlusBuildResults/EnergyPlus-d4e72c89b91b70bf6662bed2764514ad6410e3eb-x86_64-MacOS-10.9-clang.html#test

### Details
Some additional details for this issue (if relevant):
 - Platform (Mac CI)
 - Version of EnergyPlus (any)

### Checklist
Add to this list or remove from it as applicable.  This is a simple templated set of guidelines.
 - [ ] Defect file added (list location of defect file here)
 - [X] Ticket added to Pivotal for defect (development team task)
 - [ ] Pull request created (the pull request will have additional tasks related to reviewing changes that fix this defect)

**Comment by Myoldmopar** [comment]
![image](https://user-images.githubusercontent.com/849824/136090831-15476625-c897-48ca-a691-b7cc0ce6b959.png)

The zone equipment array has a sizing method array member, and the member is being used uninitialized.  Most of the time using zero, which is actually not the right code path, but the unit test was written to match this invalid value.  Every once in a while the uninitialized memory is nonzero and causes the code to go down the wrong code path.  Sheesh.

I found this during an enum pass, where the integer was changed to an enum class, and I initialized it with a different value, and the unit test started failing locally.  Debugging in develop I found the SizingMethod was allocated, but the numeric values were all nonsense.

Fix will be included in an upcoming PR.
